plugin-maven/README.md

```diff
@@ -115,6 +115,60 @@ By default, all files matching `src/main/java/**/*.java` and `src/test/java/**/*
 </configuration>
 ```
 
+<a name="format"></a>
+
+## Applying to custom sources
+
+By default, no Ant-Style include patterns are defined.  Each element under `<format>` is a step, and they will be applied in the order specified.  Every step is optional, and they will be applied in the order specified.
+
+```xml
+<configuration>
+  <format>
+    <includes>
+      <!-- include all property files in "resource" folders under "src" -->
+      <include>src/**/resources/**/*.properties</include>
+    </includes>
```

Comment:
Does this section define default includes?
I think elements from `<format>` should be applied only to files defined by `<java>` and `<scala>`. So maybe we do not even need `<includes>` and `<excludes>` in `<format>`?

Reply:
No the `<format>` section does not define default includes, as you don't know beforehand what its used for see above (properties, sql ,..).